.github/workflows/build.yml

```diff
@@ -234,6 +236,7 @@ jobs:
         -DBATTERY=ON
         -DBROADCAST=ON
         -DHID=ON
+        -DKEYFINDER=ON
```

Comment:
Why not enable this by default in CMakeLists.txt?

Reply:
Explicit is better than implicit here. We should enable them explicitly, so that we get an error message in case we mess up our package detection in CMakeLists.txt. Otherwise me might accidently ship Mixxx with important features disabled, because we didn't notice that detection failed.
The defaults are only for our user's convenience.